fitbenchmarking/utils/default_options.ini

```diff
@@ -92,11 +92,12 @@ use_errors: yes
 #              have the top ranking (brightest) and values over 3 will show as
 #              the worst ranking (deep red)
 # default thresholds are 1.1, 1.33, 1.75, 3, and inf
-colour_scale: 1.1, ranking-top-1
-              1.33, ranking-top-2
-              1.75, ranking-med-3
-              3, ranking-low-4
-              inf, ranking-low-5
+colour_scale: 1.1, ranking-top-1, #fef0d9
+              1.33, ranking-top-2, #fdcc8a
+              1.75, ranking-med-3, #fc8d59
+              3, ranking-low-4, #e34a33
+              inf, ranking-low-5, #b30000
+HTML_COLOUR_SCALE = ['', '', '', '', '']
```

Comment:
What is HTML_COLOUR_SCALE for now? I can't see any reference to it. If it is needed it will need a comment to explain usage and should use the same list style as the other lists.

Reply:
Thanks, this has been removed